_articles/de/starting-a-project.md

```diff
@@ -18,16 +18,9 @@ Sie denken also über einen Sprung in die Open-Source-Welt nach? Herzlichen Glü
 
 Ein Open-Source-Projekt kann von **jedem Menschen für jeden Zweck verwendet, inspiziert, modifiziert und weiterverbreitet werden.** Diese Rechte werden [mittels einer Open-Source-Lizenz durchgesetzt](https://opensource.org/licenses).
 
-Open Source ist leistungsstark, weil es Akzeptanzbarrieren senkt und Ideen schneller verbreiten lässt.
+Open Source ist so mächtig, weil es Barrieren für Nutzung und Zusammenarbeit senkt, und Leuten somit ermöglicht, Ideen schneller zu verbreiten, sowie Dinge schneller zu verbessern. Zudem ermöglicht es den Benutzer\*inn\*n, ihr eigenes Computing besser unter Kontrolle zu behalten, als mit proprietären Lösungen. Ein Unternehmen, das Open-Source-Software verwendet, kann bspw. jemanden anheuern, um maßgeschneiderte Verbesserungen an der Software vorzunehmen, anstatt völlig abhängig zu sein von den Produktentscheidungen des Anbieters.
```

Comment:
Delete comma after **senkt** and after **ermöglicht**
I think the commas there are unnecessary.
I would change **bspw.** to **beispielsweise**
I would change **anheuern** to **einstellen**
I would change **... anstatt von den Produktentscheidungen des Anbieters abhängig zu sein.**

Reply:
At least [one `,` needs to stay](https://www.duden.de/sprachwissen/rechtschreibregeln/komma#D121), IMHO.

I like `anheuern`, because it includes freelance work. Insinuating regular employment as the default, might be strategically more useful, though ;-)